.secrets.baseline

```diff
@@ -79,35 +79,35 @@
         "hashed_secret": "6955d0539c7ae97361ab63d21bc2bb730edbce7a",
         "is_secret": false,
         "is_verified": false,
-        "line_number": 3903,
+        "line_number": 3904,
```

Comment:
lol detect-secrets thinks our tests are some kind of cryptographic token... I don't know whether to laugh at detect-secrets or cry at our tests 🤦